reporting-guide.md

```diff
@@ -16,6 +16,19 @@ Before you can use Analytics APIs, you need to obtain authentication and receive
 
 The `/reports` endpoint description is shown in our [Swagger UI](https://adobedocs.github.io/analytics-2.0-apis/). Use the Swagger UI to see endpoint summaries, available methods, parameters, example values, models, and status codes, and to try out the API.
 
+## Partial Responses (206 Status Code)
+
+A 206 status code indicates a partial response.  This means there were some columns in the reporting response that has errors.  These errors may be any of the following:
```

Comment:
This means there were some columns in the reporting response that **have** errors.

Reply:
Fixed